pkgs/Shopify/toxiproxy/registry.yaml

```diff
@@ -0,0 +1,21 @@
+packages:
+  - type: github_release
+    repo_owner: Shopify
+    repo_name: toxiproxy
+    asset: toxiproxy_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz
+    description: "A TCP proxy to simulate network and system conditions for chaos and resiliency testing"
```

Comment:
```suggestion
    description: A TCP proxy to simulate network and system conditions for chaos and resiliency testing
```

https://aquaproj.github.io/docs/reference/registry-style-guide#remove-unneeded-quotes-of-strings